src/common/util.h

```diff
@@ -58,6 +58,8 @@ extern unsigned long long Mmap_align;
 #define MMAP_ALIGN_UP(size) (((size) + Mmap_align - 1) & ~(Mmap_align - 1))
 #define ADDR_SUM(vp, lp) ((void *)((char *)(vp) + lp))
 
+#define alignof(t) offsetof(struct {char c; t m; }, m)
```

Comment:
I would rather prefer including stdaling.h, and using alignof from there. Only Visual C++ without the LLVM frontend is a case where no C11 might be available, and for that case we can use this:
https://msdn.microsoft.com/en-us/library/45t0s5f4.aspx

Or at least use underscores to prefix the symbols inside the macro.
I tried this macro, and doesn't compile if there is a local variable named m, or c present.